statsmodels/tsa/stattools.py

```diff
@@ -574,6 +574,65 @@ def pacf_yw(x, nlags=40, method='unbiased'):
     return np.array(pacf)
 
 
+def pacf_burg(x, nlags=None, demean=True):
+    """
+    Parameters
+    ----------
+    x : array-like
+        Observations of time series for which pacf is calculated
+    nlags : int, optional
+        Number of lags to compute the partial autocorrelations.  If omitted,
+        uses the smaller of 10(log10(nobs)) or nobs - 1
+    demean : bool, optional
+
+    Returns
+    -------
+    pacf : ndarray
+        Partial autocorrelations for lags 0, 1, ..., nlag
+    sigma2 : ndarray
+        Residual variance estimates where the value in position m is the
+        residual variance in an AR model that includes m lags
+
+    See also
+    --------
+    statsmodels.tsa.stattools.pacf
+
+    References
+    ----------
+    Brockwell, P.J. and Davis, R.A., 2016. Introduction to time series and
+        forecasting. Springer.
+    """
+    x = np.squeeze(np.asarray(x))
+    if x.ndim != 1:
+        raise ValueError('x must be 1-d or squeezable to 1-d.')
+    if demean:
+        x = x - x.mean()
+    nobs = x.shape[0]
+    p = nlags if nlags is not None else min(int(10 * np.log10(nobs)), nobs - 1)
+    if p > nobs - 1:
+        raise ValueError('nlags must be smaller than nobs - 1')
+    d = np.zeros(p + 1)
+    d[0] = 2 * x.dot(x)
+    phi = np.zeros(p + 1)
```

Comment:
I don't know if we're still trying for descriptive naming, but could be `arcoefs` or something if we are.

Reply:
Good idea.